.github/workflows/tests.yml

```diff
@@ -16,22 +16,20 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-
 jobs:
+  define-matrix:
```

Comment:
I think it would be good to add a comment what this actually does - in other cases it is clear it does all branches (eq. Rawhide, branched Fedora & RHEL branches) but here it is not clear to me. From the code being removed it looks like it does Rawhide & branched, but that would not be evident to me from just the comment, so I suggest we expand the comment a bit. :)